rust/Cargo.toml.in

```diff
@@ -29,12 +29,12 @@ num-derive = "0.2"
 num-traits = "0.2"
 widestring = "0.4"
 
-der-parser = "3.0"
-kerberos-parser = "0.4"
+der-parser = "4.0.0-alpha3"
+kerberos-parser = { version = "0.4", git = "https://github.com/sharksforarms/kerberos-parser", branch = "upgrade-der-parser" }
```

Comment:
are these "overrides" still needed?

Reply:
They are not and have been removed. I accidentally force pushed on this branch instead of a v8 branch, it has been corrected.

Reply:
When I check out the PR I still get this change. Can you do a v8?